_posts/2020-06-02-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md

```diff
@@ -111,10 +111,6 @@ free to reach out to us with any question you might have and stay tuned for more
 
 ---
 Follow Sourcify on [Twitter](https://twitter.com/SourcifyEth). 
```

Comment:
No `<br>`?

Reply:
Same here, see my other comment. I think we either need `<br>`or bullets or some other way to create a line break, otherwise it all appears in one line.

Reply:
We need an empty new line. Would avoid using HTML tags in markdown. Anyway fixed.